lib-extra/build.gradle

```diff
@@ -22,6 +22,5 @@ dependencies {
 }
 
 // we'll hold the core lib to a high standard
-findbugs {
-	reportLevel = 'low' // low|medium|high (low = sensitive to even minor mistakes)
+findbugs { reportLevel = 'low' // low|medium|high (low = sensitive to even minor mistakes)
 }
```

Comment:
Hmm, that's a shame.

Could we work around this problem with the following manual formatting?
```gradle
findbugs { reportLevel = 'low' } // low|medium|high (low = sensitive to even minor mistakes)
```